src/ApiClient.php

```diff
@@ -104,8 +95,7 @@ public function validateTranslationRequest(TranslationRequest $translationReques
         $endpoint = $this->container->get('validate');
         return $endpoint
             ->setToken($this->getToken())
-            ->setTranslationRequest($translationRequest)
```

Comment:
I got rid of these setters/getters in favor of method params (like `validateTranslationRequest($translationRequest)`). This way, the code is significantly reduced.